app/views/layout.erb

```diff
@@ -40,6 +40,7 @@
     <img src="/gorilla_banner.png" alt="gorilla banner" />
   </div>
       <div id="content">
+        <%= flash[:message] %>
```

Comment:
we dont want this to render every time. Can you surround this in control flow like 
if flash[:message]
  flash[:message]
end
that way it would only show up when we put something in the flash. else we will be rendering nil sometimes and I am not sure if that is bad.